### ISSUE TITLE
define stream throws error if it is declared inside a class

### ISSUE THREAD

**Issue by gustason** [BalticAmadeus/AblFormatter#225]
For example:
``` abl
class FunkyLogger: 
    define stream myLittleLogger.
end class.
```
gets parsed as this:
![image](https://github.com/user-attachments/assets/9f8ff20c-8169-485d-87ca-defa8d6f3307)

**Comment by eglekaz** [comment]
Registered: https://github.com/usagi-coffee/tree-sitter-abl/issues/94

**Comment by eglekaz** [comment]
Fixed with version 0.0.44 of tree-sitter-abl
